### DISCUSSION TITLE
How change the timezone to Asia/Shanghai

### DISCUSSION THREAD

**Discussion by wer860612** [apache/airflow#26614] [General]
I want to modify the UTC time zone to Asia/Shanghai time zone in docker, 
I already modify the file /home/airflow/.local/lib/python3.8/site-packages/airflow/utils/timezone.py & /home/airflow/.local/lib/python3.8/site-packages/airflow/utils/sqlalchemy.py
I use datetime.now() to get time , but the time print still in utc timezone

[output]
2022-09-23 03:03:35.573827

[correct]
2022-09-23 11:03:35.573827

**Apache Airflow version**
2.2.1 - python3.8

**Comment by potiuk** [comment]
Googled it, so it might be not good but if you look in the internet how to change timezone in linux you might find proper recipes: https://linuxize.com/post/how-to-set-or-change-timezone-in-linux/

This is a question of linux/container timezone, not python timezone.

**Reply by wer860612** [reply]
Hi @potiuk
The time I printed was correct. but the airflow logging time was adding 8 hours.
